kale/utils/csv_logger.py

```diff
@@ -223,7 +223,7 @@ def print_scores(
         split="Validation",
         stdout=True,
         fdout=None,
-        print_func=print,
+        print_func=logging.info,
```

Comment:
Shall we keep using `print` here? If yes, we could since it is meant to be a **utility** function, like in `print.py`.

I checked that `print_scores` is **only** called in `main.py` of our two examples and it should have no those negative aspects of keeping `print` in core API.

Reply:
Or shall we change that `end=` to something else?

Reply:
We can remove the `end=` and do a minor change. I will change and commit.
If we use `print`, the output is 
```Validation DAN       38.7% +- 5.25 (62 runs)		25.0% +- 6.27 (62 runs)		nan% +- nan (62 runs)```

If `logging.info`, the output is 
```@2021-01-31 17:53:40,339 root [INFO] - (Validation DAN	 39.1% +- 5.10 (64 runs)		25.0% +- 6.17 (64 runs)		nan% +- nan (64 runs))```

Reply:
Great. I've turned the auto-merge on so once you make the changes and approve, this PR will merge automatically. Many thanks.